### DISCUSSION TITLE
Few beginner questions

### DISCUSSION THREAD

**Discussion by igorwojda** [awawa-dev/HyperHDR#830] [General]
Hello

I am getting started with building my own ambilight and I wonder about few things.

But first my setup: 75 inch TV (requires around 4-5m of LEDs)

1. Which RGBW LEDs are recommended these days (fast refresh rate is important to me)
2. Can fast refresh rate be achieved with single segment and proper LEDs or multisegment is a way to go?
3. Considering the above which LED model and LED Controller model would you recommend?
4. I have seen video where LED controller has to be flashed, so it can be controlled by HyperHDR (https://youtu.be/4jkwFsMkKwU?t=757). Video is older one so I wonder how this is done these days?

**Comment by awawa-dev** [comment]
Hi
1 cold white sk6812 RGBW is still recommended. BTF-Lighting advertises that they have a refreshed version that runs dataline at 3.3V, but I have not seen any official technical specifications nor do I have a sample to verify their claims.
2 sample results can be found on the blog and on the project websites: HyperSerialPico/ESP32 or HyperSPI so you can decide yourself.
3 rp2040 with built-in level shifter and HyperSerialPico. HyperSPI also should work with Pico but I consider it as an experimental support since I havent received feedback from other Pico users. But it works for me.
4 Yes, it's quite old. I recommend a wired LED driver instead of a wireless one.

**Comment by igorwojda** [comment]
Amazing thx @awawa-dev.
BTW opened [small PR](https://github.com/awawa-dev/HyperSerialPico/pull/23) in you repo to clear things up.
